chrome/content/otr.js

```diff
@@ -276,6 +284,18 @@ var otr = {
     return human.readString();
   },
 
+  base64encode: function(data, dataLen) {
+    let buf = ctypes.char.array(Math.floor(((dataLen+2)/3)*4 + 1))();
+    let size = libOTR.otrl_base64_encode(buf, data, dataLen);
+    return buf.readString();
+  },
+
+  base64decode: function(base64data) {
+    let buf = ctypes.unsigned_char.array(Math.floor((base64data.length+3)/4)*3)();
```

Comment:
If you're going to floor, you need to +1 like above, no? But why not ceil for both?